content/enterprise_influxdb/v1.9/administration/cluster-commands.md

```diff
@@ -161,15 +161,15 @@ Error: authorization failed.
 
 Adds a data node to a cluster.
 By default, `influxd-ctl` adds the specified data node to the local meta node's cluster.
-Use `add-data` instead of the [`join` argument](#join) when performing a [production installation](/enterprise_influxdb/v1.9/production_installation/data_node_installation/) of an InfluxDB Enterprise cluster.
+Use `add-data` instead of the [`join` argument](#join) when [installing](/enterprise_influxdb/v1.9/installation/data_node_installation/) an InfluxDB Enterprise cluster.
```

Comment:
```suggestion
Use `add-data` instead of the [`join` argument](#join) when [installing a data node](/enterprise_influxdb/v1.9/installation/data_node_installation/) an InfluxDB Enterprise cluster.
```